UVAtlas/isochart/isochartengine.h

```diff
@@ -122,6 +122,7 @@ class CIsochartEngine : public IIsochartEngine
         float& MaxChartStretchOut,
         uint32_t* pFaceAttributeIDOut);
 
+    HRESULT ParameterizeChartsInHeapParallelized(bool bFirstTime, size_t MaxChartNumber);
```

Comment:
Again ``#ifdef _OPENMP`` around this method's prototype.

Reply:
done